.changeset/empty-llamas-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/vitest-pool-workers": patch
+---
+
+Fix `"module": "..."` resolution in vitest-pool-workers
```

Comment:
This is a very vague changeset and not really very actionable by the person reading the released change log. (Also the format is slightly off).

Can I propose we fill it out a bit:

```suggestion
fix: Support importing ES modules from libraries that do not correctly provide `"type"="module"` not use `.mjs` extensions

The toucan-js library has an entry point of `"module": "dist/index.esm.js"`. This file does not use the standard `.mjs` extension, nor does it specify `"type"="module"`, so the resolution and loading algorithm fails to identify this file as an ES Module, defaulting to CommonJS, breaking Vitest.

Fixes #5588
```